test/extended/cli/mustgather.go

```diff
@@ -348,16 +345,29 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
 	})
 })
 
-func getPluginOutputDir(oc *exutil.CLI, tempDir string) string {
-	imageClient := versioned.NewForConfigOrDie(oc.AdminConfig())
-	stream, err := imageClient.ImageV1().ImageStreams("openshift").Get(context.Background(), "must-gather", metav1.GetOptions{})
+// getPluginOutputDir returns the directory containing must-gather assets.
+// Before [1], the assets were placed directly in tempDir.  Since [1],
+// they have been placed in a subdirectory named after the must-gather
+// image.
+//
+// [1]: https://github.com/openshift/oc/pull/84
+func getPluginOutputDir(tempDir string) string {
+	files, err := os.ReadDir(tempDir)
 	o.Expect(err).NotTo(o.HaveOccurred())
-	imageId, ok := imageutil.ResolveLatestTaggedImage(stream, "latest")
-	o.Expect(ok).To(o.BeTrue())
-	pluginOutputDir := path.Join(tempDir, regexp.MustCompile("[^A-Za-z0-9]+").ReplaceAllString(imageId, "-"))
-	fileInfo, err := os.Stat(pluginOutputDir)
-	if err != nil || !fileInfo.IsDir() {
-		pluginOutputDir = tempDir
+	dir := ""
+	for _, file := range files {
+		if file.IsDir() {
+			if dir != "" {
```

Comment:
Not sure we need this kind of check, but it sure doesn't hurt :wink:

Reply:
This is the "I'm an old-style must gather" check.  I'm not sure how many tests we still run with `oc` that old, but the outgoing `if err != nil || !fileInfo.IsDir() {` was also trying to figure out old vs. new, so keeping this check just rolls that coverage along.